homework_2_pokeapi.js

```diff
@@ -0,0 +1,64 @@
+// 6
+// Написать асинхронную функцию, которая делает запрос на
+// https://pokeapi.co/api/v2/pokemon и в консоль выводит массив полученых покемонов
+
+async function getPokemons() {
+  let response = await fetch('https://pokeapi.co/api/v2/pokemon')
+  console.log(response)
+  let result = await response.json()
+  console.log(result.results)
```

Comment:
1. why let? 
2. when you are working with async functions, you have to wrap your code into `try catch`